Makefile

```diff
@@ -16,134 +17,93 @@
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
 #*******************************************************************************
-
 ifeq ($(BOLOS_SDK),)
-$(error BOLOS_SDK is not set)
+$(error Environment variable BOLOS_SDK is not set)
 endif
 
-dummy_submodules := $(shell git submodule update --init --recursive)
+include $(BOLOS_SDK)/Makefile.defines
 
-ifeq ($(TARGET_NAME),TARGET_NANOS)
-APP_STACK_SIZE:=880
-endif
+########################################
+#        Mandatory configuration       #
+########################################
 
-include $(BOLOS_SDK)/Makefile.defines
+# Enabling DEBUG flag will enable PRINTF and disable optimizations
+#DEBUG = 1
 
-# Main app configuration
-APPNAME="Binance Chain"
-APPVERSION_M=1
-APPVERSION_N=1
-APPVERSION_P=7
+# Application name
+APPNAME = "Binance Chain"
 
-APP_LOAD_PARAMS = --appFlags 0x200 --delete $(COMMON_LOAD_PARAMS) --path "44'/714'"
+# Application version
+APPVERSION_M= 1
+APPVERSION_N= 1
+APPVERSION_P= 7
+APPVERSION = "$(APPVERSION_M).$(APPVERSION_N).$(APPVERSION_P)"
 
-ifeq ($(TARGET_NAME), TARGET_NANOS)
-ICONNAME=nanos_app_binance.gif
-else
-ICONNAME=nanox_app_binance.gif
-endif
+# Application source files
+APP_SOURCE_PATH += src
+APP_SOURCE_PATH += glyphs
```

Comment:
Is it really needed?

Reply:
I will check if it's added somewhere else already.

Reply:
`glyphs` folder not needed. I removed it.